packages/web-app-files/tests/unit/views/PublicFiles.spec.ts

```diff
@@ -1,17 +1,46 @@
+import { mount, RouterLinkStub } from '@vue/test-utils'
 import { accentuatesTableRowTest } from './views.shared'
 import PublicFiles from '@files/src/views/PublicFiles.vue'
-import { localVue } from './views.setup'
-
-/**
- * FixMe:
- * localVue is shared, no isolated testing is happening.
- * if VueRouter is applied multipleTimes it fails (as expected).
- * as temporary workaround apply it here too.
- *
- * create a localVue factory to guarantee isolation
- */
-import VueRouter from 'vue-router'
-localVue.use(VueRouter)
+import { localVue, getStore, createFile } from './views.setup'
+
+const createFolder = ({ name = '1234', canCreate = false } = {}) => ({
+  name: name,
+  canCreate: () => canCreate
+})
+
+const stubs = {
+  'list-loader': true,
+  'not-found-message': true,
+  'no-content-message': true,
+  'resource-table': true,
+  'context-actions': true,
+  pagination: true,
+  'list-info': true,
+  'router-link': RouterLinkStub
+}
+
+const $route = {
+  name: 'files-personal',
```

Comment:
This shouldn't be the route name for public files AFAIK :)

Reply:
fixed :)